examples/twittermap/web/public/javascripts/map/multi_layer.js

```diff
@@ -0,0 +1,106 @@
+angular.module('cloudberry.map')
```

Comment:
Better change the file name to `multilayer-manager.js`, which will be more meaningful.